File: Matlab/DeepGLM/nnfun/nnSumResidualSquare.m
function out = nnSumResidualSquare(y,X,W_seq,beta)
%NNSUMRESIDUALSQUARE Calculate sum of square of residuals
%
%
%   Copyright 2018 Minh-Ngoc Tran (minh-ngoc.tran@sydney.edu.au) and Nghia
%   Nguyen (nghia.nguyen@sydney.edu.au).
%   
%   http://www.xxx.com
%
%   Version: 1.0
%   LAST UPDATE: April, 2018

nnet_output = nnFeedForward(X,W_seq,beta);
out = sum((y-nnet_output).^2);
end

File: R/02_libs/nnSumResidualSquare.R
# Function to calculate sum square error of 2 vector
nnSumResidualSquare <- function(y,X,W_seq,beta){
  nnet_output <- nnFeedForward(X,W_seq,beta)        # Output vector of NN
  S <- sum((y-nnet_output)^2)
  return(S)
}
